README.md

```diff
@@ -1311,7 +1311,7 @@ _See also [Ticketing](#ticketing), [Task management/To-do lists](#task-managemen
 See **[Ticketing](#ticketing)**
 
 ### IDE/Tools
-
+- [Atheos](https://www.atheos.io) - Web-based IDE framework with a small footprint and minimal requirements, continued from Codiad. ([Source Code](https://github.com/Atheos/Atheos)) `MIT` `PHP/Javascript`
```

Comment:
- Please keep the blank line
- `PHP` is enough as it is the only server side requirement